frontend/src/metabase/query_builder/actions.js

```diff
@@ -480,13 +481,17 @@ export const SET_CARD_AND_RUN = "metabase/qb/SET_CARD_AND_RUN";
 export const setCardAndRun = createThunkAction(SET_CARD_AND_RUN, (runCard, shouldUpdateUrl = true) => {
```

Comment:
Is `runCard` a card object? I'm guessing it is based on how it's used later in the action, but it sounds more like a function name to me. (I realize this was already here but might be good to clean it up).